eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -16,7 +16,41 @@
 
   <!-- Add Javadoc suppression for any test files (ie. ones that live under src/test/java). -->
   <suppress checks="Javadoc" files="src[/\\]test[/\\]java[/\\].*.java"/>
-
+ <!-- <suppress checks="Javadoc" files="EventPosition.java"/>
```

Comment:
Why are we adding all these suppressions?

Reply:
All the suppression is going to be removed before PR is merged as I work on them .